src/components/Popups/PopupReply.tsx

```diff
@@ -69,27 +84,19 @@ export default function PopupReply({
     ...rest
 }: Props): JSX.Element {
     const popupRef = React.useRef<PopupBase>(null);
-    const popupOptions = React.useRef<Partial<Popper.Options> | null>(null);
+    const popupOptions = React.useRef<Partial<Popper.Options>>(getModifiers()); // Keep the options reference the same between renders
     const rotation = ReactRedux.useSelector(getRotation);
     const prevRotation = usePrevious(rotation);
     const scale = ReactRedux.useSelector(getScale);
     const prevScale = usePrevious(scale);
 
-    // Keep the options reference the same between renders
-    if (!popupOptions.current) {
-        popupOptions.current = {
-            ...options,
-            placement: isIE() ? 'top' : 'bottom',
-        };
-    }
-
     React.useEffect(() => {
         const { current: popup } = popupRef;
 
         if (popup?.popper && (rotation !== prevRotation || scale !== prevScale)) {
             popup.popper.update();
         }
-    }, [popupRef, rotation, scale]); // eslint-disable-line react-hooks/exhaustive-deps
+    }, [popupRef, prevRotation, prevScale, rotation, scale]);
```

Comment:
Nice catch here. Does rotate/zoom still update the reply popup position as before?

Reply:
zoom does, on rotate we disable create annotation so it disappears anyway